packages/office-ui-fabric-react/src/components/Stack/examples/Stack.Horizontal.Spacing.Example.tsx

```diff
@@ -32,20 +32,44 @@ export class HorizontalStackSpacingExample extends React.Component<{}, {}> {
       }
     });
 
+    const tokens = {
+      numericalSpacing: {
+        childrenGap: 10
+      },
+      customSpacing: {
+        childrenGap: '10%'
+      },
+      themedExtraSmall: {
+        childrenGap: 's2'
+      },
+      themedSmall: {
+        childrenGap: 's1'
+      },
+      themedMedium: {
+        childrenGap: 'm'
+      },
+      themedLarge: {
+        childrenGap: 'l1'
+      },
+      themedExtraLarge: {
+        childrenGap: 'l2'
+      }
+    };
+
     return (
-      <Stack gap={10}>
+      <Stack tokens={tokens.numericalSpacing}>
         <Stack horizontal disableShrink horizontalAlign="space-between">
           <Stack>
             <span>Numerical spacing</span>
-            <Stack horizontal className={styles.root} gap={10} padding={10}>
+            <Stack horizontal className={styles.root} tokens={tokens.numericalSpacing} padding={10}>
```

Comment:
There's still a lot of duplication going on. Once we get `extendsComponent` in we can do something like:

```tsx
const NumberStack = extendComponent(Stack, { horizontal: true, styles: { }, padding: 10, tokens: { childrenGap: 10 } });
```

Then just use:

```tsx
            <NumberStack>
              <span>1</span>
              <span>2</span>
              <span>3</span>
            </NumberStack>
```